src/main/java/xyz/groundx/caver_ext_kas/kas/kip37/KIP37.java

```diff
@@ -0,0 +1,3053 @@
+        request.setIds(Arrays.asList(ids));
+        request.setAmounts(Arrays.asList(amounts));
+
+        return tokenApi.transferTokenAsync(chainId, addressOrAlias, request, callback);
+    }
+
+    /**
+     * Pause the operations of a specified token, such as minting and creating tokens.<br>
+     * It sets a sender to contract deployer.<br>
+     * POST /v1/contract/{contract-address-or-alias}/token/pause/{token-id}
+     *
+     * <pre>{@code
+     * String contractAddress = "0x{contractAddress}";
+     * BigInteger tokenId = BigInteger.valueOf(1);
+     *
+     * Kip37TransactionStatusResponse response = caver.kas.kip37.pauseToken(contractAddress, tokenId);
+     * }</pre>
+     *
+     * @param addressOrAlias The contract address(in hex, with the 0x prefix) or alias.
+     * @param tokenId The token id to pause operation.
+     * @return Kip37TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip37TransactionStatusResponse pauseToken(String addressOrAlias, BigInteger tokenId) throws ApiException {
+        return pauseToken(addressOrAlias, tokenId, null);
+    }
+
+    /**
+     * Pause the operations of a specified token, such as minting and creating tokens.<br>
+     * It sets a sender to contract deployer.<br>
+     * POST /v1/contract/{contract-address-or-alias}/token/pause/{token-id}
+     *
+     * <pre>{@code
+     * String contractAddress = "0x{contractAddress}";
+     * String tokenId = "0x1";
+     *
+     * Kip37TransactionStatusResponse response = caver.kas.kip37.pauseToken(contractAddress, tokenId);
+     * }</pre>
+     *
+     * @param addressOrAlias The contract address(in hex, with the 0x prefix) or alias.
+     * @param tokenId The token id to pause operation.(in hex, with the 0x prefix)
+     * @return Kip37TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip37TransactionStatusResponse pauseToken(String addressOrAlias, String tokenId) throws ApiException {
+        return pauseToken(addressOrAlias, tokenId, null);
+    }
+
+    /**
+     * Pause the operations of a specified token, such as minting and creating tokens.<br>
+     * POST /v1/contract/{contract-address-or-alias}/token/pause/{token-id}<br>
+     *
+     * <pre>{@code
+     * String contractAddress = "0x{contractAddress}";
+     * BigInteger tokenId = BigInteger.valueOf(1);
+     * String sender = "0x{senderAddress}";
+     *
+     * Kip37TransactionStatusResponse response = caver.kas.kip37.pauseToken(contractAddress, tokenId, sender);
+     * }</pre>
+     *
+     * @param addressOrAlias The contract address(in hex, with the 0x prefix) or alias.
+     * @param tokenId The token id to pause operation.
+     * @param sender The account address to execute token operations.
+     * @return Kip37TransactionStatusResponse
+     * @throws ApiException
+     */
+    public Kip37TransactionStatusResponse pauseToken(String addressOrAlias, BigInteger tokenId, String sender) throws ApiException {
```

Comment:
I think for minting and pausing, there is specific role name(?)s are existed, so using `minter` and `pauser` will be more clear to understand.

Reply:
I modified it. Would you check it?